plugins/generic/recommendByAuthor/locale/pt_BR/locale.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * plugins/generic/recommendByAuthor/locale/en_US/locale.xml
```

Comment:
should be 
`plugins/generic/recommendByAuthor/locale/pt_BR/locale.xml/pt_BR/locale.xml`